backend/algorithms/recommendations.py

```diff
@@ -11,14 +11,38 @@
 
 NOTE: For sample user data, look at exampleUsers.json to get an idea of the structure
 """
-
+from .conditions import User, create_condition, CACHED_CONDITIONS_TOKENS
```

Comment:
can you import from the backend directory? So change .conditions to algorithms.conditions.

You can run from the backend directory with python3 -m algorithms.recommendations

If everything is run from the backend directory, it makes importing to other areas like the API much easier

Reply:
I added a function for loading the cached conditions and changing the paths. They are running properly by calling python -m algorithms.recommendations now.